[FEATURE] Auto-registration of NewContentElementWizard via TCA (#521)
TYPO3-Documentation/TYPO3CMS-Reference-TSconfig: Documentation/PageTsconfig/Mod/Wizards.rst

newContentElement.wizardItems
=============================

..  versionchanged:: 13.0
    New content elements added via TCA to the
    :ref:`items <t3tca:columns-select-properties-items>` of field :sql:`CType`

>     of table :sql:`tt_content` are automatically added to the newContentElement

Reviewer: ```suggestion
    of table :sql:`tt_content` are automatically added to the New Content Element
```

>     wizard. The following page TSconfig can be used to override values set via

Reviewer: ```suggestion
    Wizard. The following page TSconfig can be used to override values set via
```

>         In the new content element wizard, content element types are grouped

Reviewer: ```suggestion
        In the New Content Element Wizard, content element types are grouped
```

        together by type. Each such group can be configured independently. The
        four default groups are: `default`, `special`, `forms` and `plugins`.

        ..  versionchanged:: 13.0
            The group `common` was renamed to `default`. A permanent migration is in
            place.

        ..  confval:: removeItems
            :name: mod-wizards-newContentElement-wizardItems-removeItems
            :type: comma separated list of groups
            :Path: mod.wizards.newContentElement.wizardItems.removeItems

            ..  versionchanged:: 13.0

            With this setting one or several groups with all their content

>             elements can be hidden in the "create new content element" wizard.

Reviewer: ```suggestion
            elements can be hidden in the New Content Element Wizard.
```

            :type: string, comma-separated list of items
            :Path: mod.wizards.newContentElement.wizardItems.[group].show

            ..  versionchanged:: 13.0
                The configuration of the New Content Element Wizard has been
                changed to automatically registering the groups and elements
                from the TCA configuration.

                The previously used option to show / hide elements
                :typoscript:`mod.wizards.newContentElement.wizardItems.<group>.show` is
                not evaluated anymore.

                All configured groups and elements are automatically shown. Removing these
                groups and elements from the New Content Element Wizard can be done via
                the option :confval:`removeItems <mod-wizards-newContentElement-wizardItems-removeItems>` and
                :confval:`[group].removeItems <mod-wizards-newContentElement-wizardItems-group-removeItems>`

>                 options.

Reviewer: Duplicate? I'm unsure

Author: Nope, one is for the group, the other is in general, two different options
